guacamole/src/main/webapp/app/client/controllers/clientController.js

```diff
@@ -522,21 +539,44 @@ angular.module('client').controller('clientController', ['$scope', '$routeParams
             }
         }
 
+        // If only Ctrl-Alt-End is pressed, and we have a one keysym from each
+        // group, and one key is being released, send Ctrl-Alt-Delete.
```

Comment:
Is that "... and one key is being released ..." aspect correct?

Reply:
Yeah, this comment needed a little cleaning.